setup.py

```diff
@@ -12,8 +13,10 @@
     version = ''
 try:
     # obtain long description from README and CHANGES
-    README = open(os.path.join(here, 'README.rst')).read()
-    CHANGES = open(os.path.join(here, 'CHANGES.rst')).read()
+    with open(os.path.join(here, 'README.rst'), 'r', encoding='utf-8') as f:
+        README = f.read()
+    with open(os.path.join(here, 'CHANGES.rst'), 'r', encoding='utf-8') as f:
+        CHANGES = f.read()
```

Comment:
Looks good (other than my comment above), can you please do the same for reading `__init__.py` above?